### DISCUSSION TITLE
pull_request event not getting fired

### DISCUSSION THREAD

**Discussion by ofekot** [community/community#31484] [Pull Requests]
Hi!

When i open pull request via **github.com ui**, sometimes ci.yml workflow isn't running

**ci.yml:**
![ci](https://user-images.githubusercontent.com/25933100/187684557-1d3281cc-00ec-42d5-b05e-dfea81321772.png)

The pull requests are merging into protected branches.
to my understanding, since there is "on" pull request", it should happen on every pull request/
But in reality, the pr's are waiting for status check to be reported, but since its not occuring, the pr's just wait for the cli check forever.
![status-check](https://user-images.githubusercontent.com/25933100/187680929-d86513d0-2e43-4c19-868b-4c42a4825979.png)

so basically, since there is a requirement for the cli check to pass, but its never even running in the first place, the pr is stuck.

Since in the Checks tab of the pr i cant even see the check, it means its never taken into consideration

The solutions ive seen only explain how to force the pr to make the check, and while they sometime dont even work, I want to fix it so it"l never happen in the first place

Appreciate the help!
